lib/tapioca/cli.rb

```diff
@@ -36,28 +36,33 @@ def init
       Commands::Todo.new(
         todo_file: DEFAULT_TODO_FILE,
         file_header: true,
+        default_command_override: nil,
       ).run
 
       print_init_next_steps
     end
 
     desc "configure", "Initialize folder structure and type checking configuration"
     option :postrequire, type: :string, default: DEFAULT_POSTREQUIRE_FILE
+    option :default_command_override, type: :string, desc: "Override the default command printed on failure"
```

Comment:
I see that this is common to almost all of the commands, so I guess it is better to pull it up to a `class_option` like `--config` instead of repeating it for each command.

Reply:
While this is defined on _most_ commands, this is not used by _all_ commands. But we could indeed make it available to all so I'm not sure.